docs/content/SRS/SRS.md

```diff
@@ -163,33 +231,83 @@ _The following Software Requirements Specification for `tinymorph` is using [Vol
 
 ### 10.1 Appearance Requirements
 
-[Insert your content here.]
+LF-A1: The Tinymorph application shall adopt a unified and sophisticated visual design.
+
+Rationale: A uniform and elegant design ensures brand recognition and provides a visually pleasing experience for users, fostering greater engagement and trust in the platform.
+
+LF-A2. The application must utilize a consistent typography system across all user interfaces.
+
+Rationale: Maintaining consistency in typography ensures readability and prevents user distraction or confusion, contributing to a cohesive user experience.
 
 ### 10.2 Style Requirements
 
-[Insert your content here.]
+LF-S1. The design of the application will be minimalist, utilizing clean lines and a limited color palette.
+
+Rationale: A minimalist design emphasizes functionality and content, enhancing usability by directing the user's focus to essential elements. This approach ensures that the interface remains uncluttered and the features more accessible.
+
+LF-S2. The application must be responsive, adapting seamlessly to various device screens and orientations.
+
+Rationale: As users may access the application from different devices with varying screen sizes, responsiveness is essential to provide a consistent experience across all platforms.
+
+LF-S3. Interactive elements such as buttons and links must contrast significantly with the background to ensure visibility and accessibility.
+
+Rationale: High contrast between elements and backgrounds enhances the visibility of interactive features, making navigation intuitive and preventing user frustration.
+
+LF-S4. The user interface should facilitate smooth transitions and logical navigation pathways between different sections and features of the application.
+
+Rationale: Efficient navigation systems reduce user effort in learning how to use the application and increase overall satisfaction by minimizing the time spent on searching for functionalities.
+
+LF-S5. The application should include visual cues and feedback for user interactions to reinforce usability.
+
+Rationale: Providing immediate visual feedback for user actions confirms the system’s responsiveness, which will help users understand the application’s behavior and reduce errors.
 
 ## 11. Usability and Humanity Requirements
 
 ### 11.1 Ease of Use Requirements
 
-[Insert your content here.]
+UH-EOU1. Tinymorph shall include a session history feature that records and displays the user's most recent editing activities such as document accesses and text modifications.
+
+Rationale: This functionality streamlines user workflow by providing quick access to recent actions, which reduces the time needed for navigation and increases efficiency.
+
+UH-EOU2. The application must allow users to revise or remove their inputs after submission.
```

Comment:
hmm, I think we can include a non-invasive method, allow users to manually accept changes or use suggestion as reference should be more than enough

Reply:
Simplified the method, focused more on now allowing users to manually accept changes or use suggestions as reference.